fare/file_management/views.py

```diff
@@ -31,8 +31,15 @@ def create():
     form = RecordForm()
     # if the form is submitted and valid
     if form.validate_on_submit():
-        # we creare one contributor object with the submitted name
-        contributors = [dict(name=form.contributor_name.data)]
+        listnames = form.contributor_name.data.split(",")
+        listcontributors = []
```

Comment:
Personally, I don't really like this variable names, can we call them "list_names" and "list_contributors"?

Reply:
I'll change it